frontend/pages/views/PhotographerListView.js

```diff
@@ -99,7 +99,8 @@ export class PhotographerListView extends React.Component {
     }
 
     srcFunc(number) {
-        return `${this.props.photoListDir}/${number}_photo.jpg`;
+        // return `${this.props.photoListDir}/${number}_photo.jpg`;
```

Comment:
Please delete old code like this rather than leaving in, as we don't need it anymore.